src/Microsoft.CodeQuality.Analyzers/Core/QualityGuidelines/AvoidDuplicateElementInitialization.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Analyzer.Utilities;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Operations;
+
+namespace Microsoft.CodeQuality.Analyzers.QualityGuidelines
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+    public sealed class AvoidDuplicateElementInitialization : DiagnosticAnalyzer
+    {
+        // TODO: Determine which id to use.
+        internal const string RuleId = "CA9999";
```

Comment:
Can you please choose a diagnostic ID as per category-range mappings define here: https://github.com/mavasani/roslyn-analyzers/blob/3dd98c2cd293b2b4e2b250844c1f8c6045c91c5a/src/Utilities/DiagnosticCategoryAndIdRanges.txt? Please ensure no other analyzer in the repo is using the same ID.

Reply:
Done